modules/olmv1-updating-an-operator.adoc

```diff
@@ -7,71 +7,186 @@
 [id="olmv1-updating-an-operator_{context}"]
 = Updating an Operator
 
-You can update your Operator by manually editing your Operator's custom resource (CR) and applying the changes.
+You can update your Operator or extension by manually editing the custom resource (CR) and applying the changes.
 
 .Prerequisites
 
 * You have a catalog installed.
-* You have an Operator installed.
+* You have downloaded a local copy of the catalog file.
```

Comment:
Do we describe how to do it somewhere?

Reply:
We describe this earlier in the doc, here: https://71258--ocpdocs-pr.netlify.app/openshift-enterprise/latest/operators/olm_v1/olmv1-installing-an-operator-from-a-catalog#olmv1-finding-operators-to-install_olmv1-installing-operator